_documentation/user/flags/wings.md

```diff
@@ -5,4 +5,25 @@ type: Good
 summary: Tank can drive in air.
 ---
 
-...
+Wings flag allows the player to maneuver their tank while in the air, enabling them to move forward, backward, or turn to either side (x and y axis), all while the gravity still affects the tank midair (z axis).
+
+## Potency
+
+The server variables affecting Wings can greatly change the potency of the flag, but in general, it is considered to be one of the most powerful flags in the game.
+
+## Server variables
+
+### Usage and effect
+
+The *wingsJumpCount* is the most commonly altered server variable. While the variable defaults to 1, meaning that the user cannot perform any additional jumps (commonly referred to as *flaps*) while in the air, map creators tend to frequently change the variable to a higher value, allowing additional maneuver over the z axis as well, essentially flying through the air.
```

Comment:
I would encourage a more clinical and less opinion-based approach to descriptions. "more common", "less common", "considered", etc. are not useful documentations. Start by describing what the variable is: "number of flaps", then describe how multiple flaps affect the tank.